### ISSUE TITLE
twisted.test.test_dirdmb.DirDbmTestCase uselessly deletes its test dir at tearDown

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3319]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)| @therve reported|
|-|-|
|Trac ID|trac#3319|
|Type|defect|
|Created|2008-06-28 08:44:10Z|

This causes some intermittent failures under Windows.

Attachments:

* [dirdbm.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/7f5/7f5b1d39148b96bad58b398dac84972876abbef4/5af960f659485047bd26cf21f303f16015a555f4.diff) (495 bytes) - added by therve on 2008-06-28 08:47:22Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__3319 3319
type__defect defect
reporter__therve therve
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1214642650000000 1214642650000000
changetime__1214666726000000 1214666726000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_1"></a>|@therve removed owner|
|-|-|

Please review.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set owner to @therve|
|-|-|

good change. :)  Can you update the copyright date and re-order the imports to match more recent convention?  Then apply.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_3"></a>|@therve set status to `closed`|
|-|-|

(In [24131]) Remove useless tearDown method of twisted.test.test_dirdmb.DirDbmTestCase
because it's causing intermittent failures under windows.

Author: therve
Reviewer: exarkub
Fixes [#3319](https://github.com/twisted/twisted/issues/3319)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
